rust/operator-binary/src/airflow_controller.rs

```diff
@@ -357,18 +523,28 @@ fn build_server_rolegroup_statefulset(
             period_seconds: Some(5),
             ..Probe::default()
         };
-        container_builder.readiness_probe(probe.clone());
-        container_builder.liveness_probe(probe);
+        cb.readiness_probe(probe.clone());
+        cb.liveness_probe(probe);
+        cb.add_container_port("http", resolved_port.into());
     }
 
-    let container = container_builder.build();
+    let container = cb.build();
 
     let metrics_container = ContainerBuilder::new("metrics")
         .image(statsd_exporter_image)
         .add_container_port(METRICS_PORT_NAME, METRICS_PORT)
         .build();
 
-    let volumes = airflow.volumes();
+    let mut volumes = airflow.volumes();
+
+    volumes.push(
+        VolumeBuilder::new("config".to_string())
+            .config_map(ConfigMapVolumeSource {
+                name: Some(rolegroup_ref.object_name()),
+                ..Default::default()
+            })
+            .build(),
```

Comment:
Sorry for not giving an example... This can be further simplified :)
```suggestion
        VolumeBuilder::new("config")
            .with_config_map(rolegroup_ref.object_name())
            .build(),
```

Reply:
great - thanks!